src/GraphQL/Caching/DefaultDocumentCache.cs

```diff
@@ -15,6 +16,8 @@ public sealed class DefaultDocumentCache : IDocumentCache
         /// </summary>
         public static readonly DefaultDocumentCache Instance = new DefaultDocumentCache();
 
-        Document? IDocumentCache.this[string query] { get => null; set { } }
+        public ValueTask<Document?> GetAsync(string query) => new((Document?)null);
```

Comment:
@Shane32 Why we used explicit interface implementation here?

Reply:
Probably a mistake